dblookupext/esdtSupply/logsGetter.go

```diff
@@ -43,14 +44,14 @@ func (lg *logsGetter) getLogsBasedOnBody(blockBody data.BodyHandler) (map[string
 			return nil, err
 		}
 
-		logsDB = mergeLogsMap(logsDB, dbLogsMb)
+		logsDB = mergeLogsSlices(logsDB, dbLogsMb)
```

Comment:
this is suboptimal because we now iterate over the values 3 times (there are 3 for's). Suggestion to revert the code, operate only on maps and do a final for range when you need to return the byte slice

Reply:
removed